Jenkinsfile

```diff
@@ -152,6 +152,17 @@ pipeline {
                     }
                     post {
                         always {
+                            sh label: 'Docker status', script: 'docker ps -al'
+                            sh label: 'Dump container logs', script: '''
+                                docker logs ${BUILD_TAG_LOWER}_route_1 > logs/route.log
```

Comment:
I think that component tests doesn't use route, but fake spineroutelookup

Reply:
it's just how the container is named